doc/classes/TileSetScenesCollectionSource.xml

```diff
@@ -113,7 +113,7 @@
 			<param index="0" name="id" type="int" />
 			<param index="1" name="packed_scene" type="PackedScene" />
 			<description>
-				Assigns a [PackedScene] resource to the scene tile with [param id]. This will fail if the scene does not extend CanvasItem, as positioning properties are needed to place the scene on the TileMap.
+				Assigns a [PackedScene] resource to the scene tile with [param id]. This will fail if the scene does not extend CanvasItem, as positioning properties are needed to place the scene on the TileMapLayer.
```

Comment:
```suggestion
				Assigns a [PackedScene] resource to the scene tile with [param id]. This will fail if the scene does not extend [CanvasItem], as positioning properties are needed to place the scene on the [TileMapLayer].
```